packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -72,7 +72,11 @@ export const Components = {
         },
       },
       BarGauge: {
+        /**
+         * @deprecated use valueV2 from Grafana 8.3 instead
+         */
         value: 'Bar gauge value',
+        valueV2: 'data-testid Bar gauge value',
```

Comment:
Aha 😃 , this is the new way to keep our selectors backward compatible right? Is it maybe a good idea to add this into the [styleguide](https://github.com/grafana/grafana/blob/main/contribute/style-guides/e2e.md)?

Reply:
You're correct, if you have any feedback or suggestions I'm really appreciate it. I'd like to see/receive feedback before adding it to the styleguide though, but I totally agree thats where it belongs 👍

Reply:
I like the idea of versioning our selectors and using the deprecated annotation to be explicit is very useful, it follows the same pattern we use in some places of the protect :)